ern-cauldron-api/test/CauldronApi-test.js

```diff
@@ -98,6 +101,23 @@ describe('CauldronApi.js', () => {
     })
   })
 
+  // ========================================================== 
+  // getCauldronSchemaVersion
+  // ========================================================== 
+  describe('getCauldronSchemaVersion', () => {
+    it('should return the Cauldron schema version', async () => {
+      const schemaVersion = await cauldronApi().getCauldronSchemaVersion()
+      expect(schemaVersion).eql('1.0.0')
```

Comment:
anyway we can have a constant that states schemaVersionCurrent & schemaVersionNext ? This can be used at other parts of the app?